woodwork/tests/accessor/test_table_accessor.py

```diff
@@ -1448,3 +1448,76 @@ def test_pop_error(sample_df):
 
     with pytest.raises(LookupError, match="Column with name missing not found in DataFrame"):
         sample_df.ww.pop("missing")
+
+
+def test_accessor_drop(sample_df):
+    xfail_dask_and_koalas(sample_df)
+
+    original_columns = sample_df.columns.copy()
+    schema_df = sample_df.copy()
+    schema_df.ww.init()
+
+    single_input_df = schema_df.ww.drop('is_registered')
+    assert len(single_input_df.ww.columns) == (len(original_columns) - 1)
+    assert 'is_registered' not in single_input_df.ww.columns
+    assert to_pandas(schema_df).drop('is_registered', axis='columns').equals(to_pandas(single_input_df))
+
+    list_input_df = schema_df.ww.drop(['is_registered'])
+    assert len(list_input_df.ww.columns) == (len(original_columns) - 1)
+    assert 'is_registered' not in list_input_df.ww.columns
+    assert to_pandas(schema_df).drop('is_registered', axis='columns').equals(to_pandas(list_input_df))
+    # should be equal to the single input example above
+    assert single_input_df.ww.schema == list_input_df.ww.schema
+    assert to_pandas(single_input_df).equals(to_pandas(list_input_df))
+
+    multiple_list_df = schema_df.ww.drop(['age', 'full_name', 'is_registered'])
+    assert len(multiple_list_df.ww.columns) == (len(original_columns) - 3)
+    assert 'is_registered' not in multiple_list_df.ww.columns
+    assert 'full_name' not in multiple_list_df.ww.columns
+    assert 'age' not in multiple_list_df.ww.columns
+
+    assert to_pandas(schema_df).drop(['is_registered', 'age', 'full_name'], axis='columns').equals(to_pandas(multiple_list_df))
+
+    # Drop the same columns in a different order and confirm resulting DataFrame column order doesn't change
+    different_order_df = schema_df.ww.drop(['is_registered', 'age', 'full_name'])
+    assert different_order_df.ww.schema == multiple_list_df.ww.schema
+    assert to_pandas(multiple_list_df).equals(to_pandas(different_order_df))
+
+
+def test_accessor_drop_rows(sample_df):
+    xfail_dask_and_koalas(sample_df)
```

Comment:
Remove?

Reply:
oh oops--was thinking we might want to test the extra pandas drop parameters, but not necessary for this pr